tables/Makefile.am

```diff
@@ -432,6 +432,12 @@ $(fuzzing_yml): $(srcdir)/Makefile.am
 
 all-local: $(fuzzing_yml)
 
+# tables that use macros need to be expanded
```

Comment:
Shouldn't the `*.in` files in the `EXTRA_DIST`?

Reply:
Yes, you may add them. I just ran `make check` for now.

Reply:
Thanks